test/cover/checkresult.py

```diff
@@ -33,23 +34,40 @@ def __init__(self, fname, must_exist=True):
 
     def __call__(self, key, value, tol=0.0):
         """Assert value == dict[key] unless value is a hoc_Vector.
-        For the Vector case, assert the maximum element difference is <= tol.
+        For the Vector case, assert the maximum relative element difference is <= tol.
         If the key does not exist add {key:value} to the dict.
         """
 
         if key in self.d:
             if type(value) == type(h.Vector):  # actually hoc.HocObject
-                m = h.Vector(self.d[key]).sub(value).abs().max()
-                if m > tol:
```

Comment:
Note that this changes the tolerance from being absolute to being relative. This wasn't the first change I made, so it's possible some tolerances are now a little looser than they need to be as a result.